apps/api/v2/src/modules/destination-calendars/inputs/destination-calendars.input.ts

```diff
@@ -0,0 +1,26 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { Expose } from "class-transformer";
+import { IsString } from "class-validator";
+
+export class DestinationCalendarsInputBodyDto {
+  @IsString()
+  @Expose()
+  @ApiProperty({
+    example: "apple_calendar",
+    description: "The calendar service you want to integrate, as returned by the /calendars endpoint",
+    type: "string",
+    required: true,
+  })
+  readonly integration!: string;
```

Comment:
This should be an enum validated using @IsEnum decorator - apple_calendar and same for google and miscrosoft calendars. Right now people can pass any string.

Reply:
Maybe instead of calling it integration we can call it `calendar` since we would not have any other type of integrations other than calendars.